storage/pruning/pruningStorer.go

```diff
@@ -802,6 +802,10 @@ func (ps *PruningStorer) changeEpoch(header data.HeaderHandler) error {
 	ps.activePersisters = append(singleItemPersisters, ps.activePersisters...)
 	ps.persistersMapByEpoch[epoch] = newPersister
 
+	return ps.removeOldPersistersIfNeeded(header, epoch)
+}
+
+func (ps *PruningStorer) removeOldPersistersIfNeeded(header data.HeaderHandler, epoch uint32) error {
```

Comment:
can we pass only one argument, either the epoch or the header?

Reply:
Yes